### PR TITLE
test on stable

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.0.7+1
```
Comment (natebosch) [inline CHANGELOG.md]: 1.0.8

Comment (kevmoo) [reply CHANGELOG.md]: yup. fixed

**Review by jakemac53 — APPROVED** [review]

File: lib/src/utils_isolate_stub.dart [hunk 2]
```diff
@@ -2,5 +2,7 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'dart:async';
```
Comment (lrhn) [inline lib/src/utils_isolate_stub.dart:5]: Rather than reintroduce this import, maybe just change the SDK requirement to 2.1.0?

Comment (kevmoo) [reply lib/src/utils_isolate_stub.dart:5]: We already pushed `1.0.7` that supports `2.0.0` – and is broken. Needed to push a fix for those folks.

Now we could entertain dropping SDK `2.0.0` in v1.0.9 of this package

**Pull request merged**